execution_engine/src/storage/trie_store/operations/mod.rs

```diff
@@ -215,6 +217,74 @@ where
     }
 }
 
+pub fn missing_descendant_trie_keys<K, V, T, S, E>(
```

Comment:
Can you add a comment to this method?

Reply:
Documented in https://github.com/CasperLabs/casper-node/pull/723/commits/d06099901e08d9f87bf02f2c5b09b43b44e60e89